file_writer.py

```diff
@@ -0,0 +1,6 @@
+import pandas as pd
+
+
+def write_csv(df, parameters):
```

Comment:
```suggestion
def write_csv_file(data_frame, parameters):
```
I'll leave the method name up to you, but I'd suggest naming the 1st param in a meaningful way.

Reply:
Done.